### PR TITLE
chore: add min fraction digits to price for page title

### PR DESCRIPTION
**Pull request by aforaleka**
Screenshot 2024-06-07 at 4 24 14 PM

### CODE REVIEW

File: src/hooks/usePageTitlePriceUpdates.ts [hunk 1]
```diff
@@ -17,13 +21,17 @@ export const usePageTitlePriceUpdates = () => {
...
+  const { tickSizeDecimals } = useAppSelector(getCurrentMarketConfig, shallowEqual) ?? {};
+
   const orderbookMidMarketPrice = useAppSelector(getCurrentMarketMidMarketPrice);
 
   const price = orderbookMidMarketPrice ?? oraclePrice;
 
   useEffect(() => {
     if (id && price && isNotTablet) {
-      const priceString = price.toLocaleString(selectedLocale);
+      const priceString = price.toLocaleString(selectedLocale, {
```
Comment (tyleroooo) [inline src/hooks/usePageTitlePriceUpdates.ts:32]: Looks like everywhere else in the app we use bignumber.toFormat to format numbers. I think ideally we would make a helper function that takes a number and an OutputType and returns a string then use that function in both Output and all these places we need strings. 

I'm happy to merge this as is though and I can make a follow up pr.

**Review by tyleroooo — APPROVED** [review]

File: src/hooks/usePageTitlePriceUpdates.ts [hunk 2]
```diff
@@ -1,29 +1,35 @@
...
+  const { tickSizeDecimals } = useAppSelector(getCurrentMarketConfig, shallowEqual) ?? {};
+
   const orderbookMidMarketPrice = useAppSelector(getCurrentMarketMidMarketPrice);
 
   const price = orderbookMidMarketPrice ?? oraclePrice;
 
   useEffect(() => {
     if (id && price && isNotTablet) {
-      const priceString = price.toLocaleString(selectedLocale);
+      const priceString = MustBigNumber(price).toFixed(tickSizeDecimals ?? SMALL_USD_DECIMALS);
```
Comment (tyleroooo) [inline src/hooks/usePageTitlePriceUpdates.ts:32]: Does this add commas to big numbers like bitcoin price?

Comment (aforaleka) [reply src/hooks/usePageTitlePriceUpdates.ts:32]: ah right - with go back to locale string, maybe theres a different setting

Comment (aforaleka) [reply src/hooks/usePageTitlePriceUpdates.ts]: lol looks like i need to set both min/max to match the decimal points

**Review by tyleroooo — APPROVED** [review]

**Pull request merged**